.byebug_history

```diff
@@ -1,4 +1,21 @@
 exit
+Order.find_by(status: "paid")
+Order.all
+Order.where(status: self.statuses["paid"])
+Order.where(status: statuses["paid"])
+Order.statuses
+Order.all
+exit
+Order.statuses
+Order.status
+Order.where(status: 0).count
+Order.where(status: 0)
+Order.where(status: 1)
+Order.where(status: "ordered")
+Order.all.count
+Order.where(status: "paid")
+Order.all
+exit
```

Comment:
Dunno why byebug is still committing to this file, as it should be git-ignored. Lets check for typos in the name or see if it somehow got removed.